docs/changehistory/NextVersion.md

```diff
@@ -4,83 +4,52 @@ publish: false
-| @bentley/presentation-common           | @itwin/presentation-common           |
-| @bentley/presentation-frontend         | @itwin/presentation-frontend         |
-| @bentley/presentation-components       | @itwin/presentation-components       |
-| @bentley/presentation-testing          | @itwin/presentation-testing          |
-| @bentley/ui-abstract                   | @itwin/appui-abstract                |
-| @bentley/ui-components                 | @itwin/components-react              |
-| @bentley/ui-core                       | @itwin/core-react                    |
-| @bentley/ui-imodel-components          | @itwin/imodel-components-react       |
-| @bentley/ui-ninezone                   | @itwin/appui-layout-react            |
-| @bentley/ui-framework                  | @itwin/appui-react                   |
-| @bentley/ecschema2ts                   | @itwin/ecschema2ts                   |
-| @bentley/webpack-tools-core            | @itwin/core-webpack-tools            |
-| @bentley/backend-webpack-tools         | @itwin/backend-webpack-tools         |
-| @bentley/build-tools                   | @itwin/build-tools                   |
-| @bentley/eslint-plugin                 | @itwin/eslint-plugin                 |
-| @bentley/imodeljs-quantity             | @itwin/core-quantity                 |
-| @bentley/imodeljs-i18n                 | @itwin/core-i18n                     |
-| @bentley/hypermodeling-frontend        | @itwin/hypermodeling-frontend        |
-| @bentley/electron-manager              | @itwin/core-electron                 |
-| @bentley/mobile-manager                | @itwin/core-mobile                   |
-| @bentley/express-server                | @itwin/express-server                |
-| @bentley/ecschema-rpcinterface-common  | @itwin/ecschema-rpcinterface-common  |
-| @bentley/ecschema-rpcinterface-impl    | @itwin/ecschema-rpcinterface-impl    |
-| @bentley/ecschema-rpcinterface-tests   | @itwin/ecschema-rpcinterface-tests   |
-| @bentley/certa                         | @itwin/certa                         |
-| @bentley/perf-tools                    | @itwin/perf-tools                    |
-| @bentley/oidc-signin-tool              | @itwin/oidc-signin-tool              |
-| @bentley/geonames-extension            | @itwin/geonames-extension            |
-| @bentley/map-layers                    | @itwin/map-layers                    |
-| @bentley/rpcinterface-full-stack-tests | @itwin/rpcinterface-full-stack-tests |
-| @bentley/imodelhub-client-tests        | @itwin/imodelhub-client-tests        |
+iTwin.js 3.0 introduces several new features and performance enhancements to the display system, along with a collection of [learning articles]($docs/learning/display/index.md) describing its capabilities. Many of these enhancements were motivated by feedback from users. If you have a new feature you'd like to see implemented, feel free to [let us know](https://github.com/iTwin/itwinjs-core/discussions)!
```

Comment:
I believe this applies to not only display system, but all new features:

> Many of these enhancements were motivated by feedback from users. If you have a new feature you'd like to see implemented, feel free to [let us know](https://github.com/iTwin/itwinjs-core/discussions)!

Should it be moved up?

Reply:
Reorganized.